nixos/modules/module-list.nix

```diff
@@ -251,6 +251,7 @@
   ./programs/pantheon-tweaks.nix
   ./programs/partition-manager.nix
   ./programs/plotinus.nix
+  ./programs/pqos-wrapper.nix
```

Comment:
This should not contain the suffix -wrapper

Reply:
Why? You're aware that this is just literally the name of the program/project, right? https://gitlab.com/sosy-lab/software/pqos-wrapper

Reply:
Oh, then we should keep it